CHANGELOG.md

```diff
@@ -1,5 +1,243 @@
+- Updated the file to use [flow strict-local](https://flow.org/en/docs/strict/#toc-strict-local) ([903eae4915](https://github.com/facebook/react-native/commit/903eae491536a9bf1b1298ccb2e9ee546b5cc79a) by [@sidverma32](https://github.com/sidverma32))
+- Add support for "togglebutton" accessibilityRole ([da899c0cc4](https://github.com/facebook/react-native/commit/da899c0cc4372830e5ca053a096b74fff2a19cb8) by [@kacieb](https://github.com/kacieb))
+
+#### Android specific
+
+- Added null check for activity in onHostResume() ([cb0782c87e](https://github.com/facebook/react-native/commit/cb0782c87e72225a09887824155c2f99bb520a80) by [@arushikesarwani94](https://github.com/arushikesarwani94))
+- Add INFO, and MENU key event support to Android TV ([bb33c1050b](https://github.com/facebook/react-native/commit/bb33c1050ba6098a68d70055e33186d9438c4374) by [@havlasme](https://github.com/havlasme))
+- Added all autofill types to TextEdit ([d9e0ea77f0](https://github.com/facebook/react-native/commit/d9e0ea77f0111fd8400c65d68e45d54e2f84287b) by [@safaiyeh](https://github.com/safaiyeh))
+- Add support to URI keyboard type in Android ([1465c8f387](https://github.com/facebook/react-native/commit/1465c8f3874cdee8c325ab4a4916fda0b3e43bdb))
+- Add MEDIA_STOP, MEDIA_NEXT, and MEDIA_PREVIOUS event support to Android TV ([3e2bb331fc](https://github.com/facebook/react-native/commit/3e2bb331fc0974bc870b2e7bd3171e585183ed1b) by [@havlasme](https://github.com/havlasme))
+- Allow configuring ndk build architectures ([d6ed1ff58b](https://github.com/facebook/react-native/commit/d6ed1ff58b2ca4d1c8b45416e56fa1da75633c07) by [@janicduplessis](https://github.com/janicduplessis))
+- Added support for accessibility role of "list" for flatlist and sectioned list ([25a16123a6](https://github.com/facebook/react-native/commit/25a16123a610ae377ced23ef81ed4c03ad7d06d9) by [@anaskhraza](https://github.com/anaskhraza))
+- Add mechanism to enforce certain Views always being visible in the View hierarchy ([34903ba418](https://github.com/facebook/react-native/commit/34903ba4185acdd3ce9c5d363953ef6316406d9d) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Support for foreground ripple in Pressable ([0823f299e5](https://github.com/facebook/react-native/commit/0823f299e560efda5c0f344fcec86cf68801f4ab) by [@intergalacticspacehighway](https://github.com/intergalacticspacehighway))
+- RNTester example for adjusting text with dynamic layout. ([78caaca768](https://github.com/facebook/react-native/commit/78caaca768b8a8c0accfa1a589718bd7ac85229f) by [@yungsters](https://github.com/yungsters))
+- Support for ScrollAway native nav bars added to ReactScrollView ([0ef5beee85](https://github.com/facebook/react-native/commit/0ef5beee855afa592cc647383ba6a3ceae9cc40a) by [@JoshuaGross](https://github.com/JoshuaGross))
+
+#### iOS specific
+
+- ([8434177722](https://github.com/facebook/react-native/commit/8434177722f70a9325f9a6adf46b5315b1f4ffa4))
```

Comment:
The author changelog is on multiple lines
```
Changelog: [iOS][Added]
1. Added new primitive type "Selection" to C++
2. Added property "selection" to TextInputProps
3. Added parser for that
```

https://github.com/facebook/react-native/commit/8434177722f70a9325f9a6adf46b5315b1f4ffa4